platform/docs/docs/platform/services/data/PanelService.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 8
+sidebar_label: Panel Service
+---
+
+# Panel Service
+
+## Overview
+
+The Panel Service provides for activating/showing a panel that was registered
+via the `getPanelModule` extension method. Such panels can be either explicitly
+activated or implicitly triggered to activate when some other event occurs.
+
+## Events
+
+The following events are published in `PanelService`.
+
+| Event                 | Description                                            |
+| --------------------- | ------------------------------------------------------ |
+| ACTIVATE__PANEL       | Fires a `ActivatePanelEvent` when a particular panel should be activated.     |
+
+
+## API
+
+### Panel Activation
+
+- `activatePanel`: Fires the `ACTIVATE_PANEL` event for a particular panel (id).
+An optional `forceActive` flag can be passed that when `true` forces the
+panel to activate. Ultimately, it is up to the panel's container whether it
```

Comment:
The `forceActive` can benefit from more explanation. It is not clear what `forces to activate` mean

Reply:
Ok I will update the text. If it is still unclear perhaps you can help me come up with some wording that we can use.